card_setup.py

```diff
@@ -1,74 +0,0 @@
-import random
-import json
-from card import Card
-
-
-number_of_players = 5
-config = json.loads(open('config.json').read())
-
-def market_setup(number_of_players):
-
-    """returns 2 lists, the first 9 dark cards for the starting marketplace and the rest of the deck ready for play"""
-    
-    # turn each card into a card object
-    cards = [Card(c) for c in config.get('cards')]
```

Comment:
All of this is now in the class method to create the initial deck in the Market class